Documentation/proposals/202405-scrapeconfig-graduation.md

```diff
@@ -77,6 +77,28 @@ The following is the list of Service Discoveries which we want to support before
 - *`scaleway_sd_config`*
 - *`ionos_sd_config`*
 
+If we were to categorize the Service Discoveries based on the amount of effort we are willing to put in maintaining them:
+
+**Tier-1:**
+Deploying the Operator to Kubernetes is our sole use case. So it is logical that we want to provide maximum support to any service discovery that fundamentally fits into the a Kubernetes cluster.
+- Kubernetes Service Discovery
+- File Service Discovery
+- Static config
+- DNS Service Discovery
+- HTTP Service Discovery
+
+**Tier-2:**
+Deploying the Operator to Kubernetes is our sole use case, so it is logical to extend support for cloud providers who provide a managed Kubernetes service.
+- Azure Service Discovery
+- EC2 Service Discovery
+- Consul Service Discovery
+- Lightsail Service Discovery
+- DigitalOcean Service Discovery
+- Kuma Service Discovery
+- OVHCloud Service Discovery
+- Scaleway Service Discovery
+- Ionos Service Discovery
```

Comment:
I would add OpenStack to the list since there are some integration points with Kubernetes. Also Consul isn't directly tied to Kubernetes but it's a widely used solution so it makes sense to include it in the list.